assets/aml-benchmark/components/src/aml_benchmark/batch_output_formatter/result_converters.py

```diff
@@ -152,11 +165,24 @@ def _get_oss_output_token(self, result: Any, perf_metrics: Any) -> Tuple[int, in
         input_parameters = ResultConverters._get_oss_input_parameters(result)
         return input_parameters.get("max_new_tokens", perf_metrics.get('output_token_count', -1))
 
+    def _get_additional_columns_data(self, data: Dict[str, Any]) -> Dict[str, Any]:
+        additional_columns_data = {}
+        if self._additional_columns:
+            for k in self._additional_columns:
+                if k in data.keys():
```

Comment:
if k column is not in data, a user error should be thrown, right?

Reply:
or just log this fact as a warning and continue?

Reply:
Please see code below.